src/Entity/Notification.php

```diff
@@ -135,4 +145,9 @@ public function setAffectation(?Affectation $affectation): self
 
         return $this;
     }
+
+    public function getHistoryRegisteredEvent(): array
```

Comment:
Finalement je ne voyais pas l’intérêt d'historiser la création de notif qui spam la table lors de mes tests.
Par contre les suppression en mode batch ne sont pas historiser c'est pas plus mal (car ne passe pas par l'entity manager mais par un DQL directement  `$qb->delete()` )

Reply:
C'est vrai que si on n'a qu'une moitié des trucs, c'est pas hyper utile, si ?

Reply:
C'est noté je supprime l'historisation sur cet entité